.github/workflows/quality.yml

```diff
@@ -0,0 +1,107 @@
+name: Code quality with Pylint & iSort
+
+on:
+  # Trigger workflow on pull request, but only for PRs merging into master and develop branches
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches:
+      - master
+      - develop
+
+jobs:
+  quality:
+
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: technote-space/get-diff-action@v3
+      with:
+        SUFFIX_FILTER: .py
+        DOT: '...'
+        DIFF_FILTER: 'AMRC'
+
+    - name: Set up python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 2.7
+
+    - name: Install OS packages
+      run: |
+        chmod +x philu_actions/scripts/dependency_downloader.sh
+        sudo bash philu_actions/scripts/dependency_downloader.sh
+
+    - name: Install edx-platform requirements
+      run: |
+        pip install -r requirements/edx/testing.txt
+        pip install -r requirements/philu/testing.txt
+
+    - name: Analysing code with iSort
+      id: isort-report
+      continue-on-error: true
+      run: |
+        isort ${{ env.GIT_DIFF }} --diff 2>&1 |tee report_isort.txt
```

Comment:
Let's please revisit the indentation, May be a space after `>` and `|` is required.

Reply:
Added space after `|`, however `2>&1` is normally written [as is](https://unix.stackexchange.com/questions/20469/difference-between-21-output-log-and-21-tee-output-log).